initialize.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+set -e
+
+# Check if docker is installed
+if ! [ -x "$(command -v docker)" ]; then
+  echo 'Error: docker is not installed.' >&2
+  # Ask if user wants to install docker
+  read -p "Do you want to install docker? [y/n] " -n 1 -r
+  echo
+  if [[ $REPLY =~ ^[Yy]$ ]]; then
+    # Install docker
+    curl -fsSL https://get.docker.com -o get-docker.sh
+    sh get-docker.sh
+    # Check if docker is installed
+    if ! [ -x "$(command -v docker)" ]; then
+      echo 'Error: Could not install docker.' >&2
+      exit 1
+    fi
+  else
+    exit 1
+  fi
+fi
+
+# Check if docker-compose is installed
+if ! [ -x "$(command -v docker-compose)" ]; then
+  echo 'Error: docker-compose is not installed.' >&2
+  # Ask if user wants to install docker-compose
+  read -p "Do you want to install docker-compose? [y/n] " -n 1 -r
+  echo
+  if [[ $REPLY =~ ^[Yy]$ ]]; then
+    # Install docker-compose
+    sudo curl -L "https://github.com/docker/compose/releases/download/1.29.2/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose
+    sudo chmod +x /usr/local/bin/docker-compose
+    sudo ln -s /usr/local/bin/docker-compose /usr/bin/docker-compose
+    # Check if docker-compose is installed
+    if ! [ -x "$(command -v docker)" ]; then
+      echo 'Error: Could not install docker-compose.' >&2
+      exit 1
+    fi
+  else
+    exit 1
+  fi
+fi
+
+# Check if python is installed
+if ! [ -x "$(command -v python)" ]; then
+  echo 'Error: python is not installed.' >&2
+  exit 1
+fi
+
+# Check if pip is installed
+if ! [ -x "$(command -v pip)" ]; then
+  echo 'Error: pip is not installed.' >&2
+  exit 1
+fi
```

Comment:
we could add the installation/update of pip

Reply:
> we could add the installation/update of pip

@mlodic Same apt dependency as above

Reply:
ok

also noticed that it should be pip3 too here